src/org/gk/model/InstanceUtilities.java

```diff
@@ -1394,4 +1396,62 @@ else if (instance.getSchemClass().isa(ReactomeJavaConstants.Pathway))
 	//public static String encodeLineSeparators(String text) {
 	//    return text.replaceAll(FileAdaptor.LINE_END + "", "<br>");
 	//}
+
+     /**
+      *
+      * @param id
+      * @return
+      */
+     public static boolean isDrug(Long id) {
+         if (id == null)
+             return false;
+
+         XMLFileAdaptor fileAdaptor = PersistenceManager.getManager().getActiveFileAdaptor();
```

Comment:
The rendering may be used for diagrams loaded from the database directly (e.g. Select an PathwayDiagram instance in the schema view in the db browser). This code will not work for this use case.

Reply:
Fixed in [c554e86](https://github.com/reactome/CuratorTool/pull/26/commits/814ef7e7024ae03dda654335352a3204ec554e86).